### DISCUSSION TITLE
WAF打开XSS 防御或SQL 注入防御后,alist无法上传文件

### DISCUSSION THREAD

**Discussion by chenmozhijin** [1Panel-dev/1Panel#5101] [Q&A]
### 1Panel 版本

v1.10.7-lts

### 问题描述

![1](https://github.com/1Panel-dev/1Panel/assets/110812055/5427937e-6369-4041-adaa-b3c8a7da50da)
![2](https://github.com/1Panel-dev/1Panel/assets/110812055/455fba28-7c41-4936-872c-09bc995f514f)
![3](https://github.com/1Panel-dev/1Panel/assets/110812055/0b6c6b73-bd77-48a7-9cb2-a11015abbff6)

### 重现步骤

在alist中上传文件

### 期待的正确结果

在alist中成功上传文件

### 附加信息

req_log.db:
![4](https://github.com/1Panel-dev/1Panel/assets/110812055/ce71d35e-9d5f-4dcd-bc6b-726255f7f66c)

**Comment by zhengkunwang223** [comment]
你可以给这个接口加白名单

**Comment by hnuzhoulin** [comment]
最后一个图的日志是必须专业版才有吗？我发现也会拦截seafile的同步接口，不是一直，重新手动同步之后会好一阵子，然后又会被拦截，导致403

**Reply by chenmozhijin** [reply]
这是我从waf的数据库提取的内容,我是社区版，也没找waf的日志在哪里

**Reply by hnuzhoulin** [reply]
如何访问这个数据库呀？

**Reply by zhengkunwang223** [reply]
专业版有拦截日志

**Reply by hnuzhoulin** [reply]
> 这是我从waf的数据库提取的内容,我是社区版，也没找waf的日志在哪里

感谢，但是居然没看到拦截seafile相关的记录
